pkg/kev/converter/kubernetes/project_service_test.go

```diff
@@ -291,20 +269,29 @@ var _ = Describe("ProjectService", func() {
 		})
 
 		Context("when deploy block `mode` defined as `global` and workload type is different than DaemonSet", func() {
-
 			projectWorkloadType := config.StatefulsetWorkload
 
-			BeforeEach(func() {
-				labels = composego.Labels{
-					config.LabelWorkloadType: projectWorkloadType,
-				}
+			JustBeforeEach(func() {
```

Comment:
No sure why we started using `JustBeforeEach`?

Reply:
because we need to change the extensions and in this case the tests needs to override the creation of the project service since it can't be updated easily

Reply:
I see.

Personally, to keep spec files simple, I try to only use just one `JustBeforeEach` for the entire spec file. This keeps things readable and easier to understand.

Do you think multiple `JustBeforeEach` statements are code small signalling that we need to split this test file into smaller test files?

Reply:
Yep, I think so for sure, don't think this is the right time to do it though because of the big refactor in progress but I will definitely add it to the list